lauren_vicknair/homework/calculator.rb

```diff
@@ -0,0 +1,35 @@
+puts "What do want from me?"
```

Comment:
This is good! But maybe you can prompt your user better by including the choices "add","subtract","multiply","divide" in your puts statement!